docs/core-env/create-custom-compute-resources.md

```diff
@@ -37,23 +37,25 @@ Content-Type: text/cloud-config; charset="us-ascii"
 packages:
 - jq
 - btrfs-progs
-- python27-pip
 - sed
 - wget
+- unzip
 # add more package names here if you need them
 
 runcmd:
-- pip install -U awscli boto3
```

Comment:
Need to keep `python-pip` and `boto3` as it's a dependency for ebs-autoscale.  That said, might be able to move that there.

Reply:
Pushed an update to awslabs/amazon-ebs-autoscale that removes the dependency on python, so this is fine now.